res/css/views/auth/_AuthBody.scss

```diff
@@ -146,3 +146,11 @@ limitations under the License.
 .mx_AuthBody_spinner {
     margin: 1em 0;
 }
+
+@media only screen and (max-width: 480px) {
+    .mx_AuthBody {
+        width: auto;
+        max-width: 500px;
+        padding: 10px;
```

Comment:
Hmm, I swear I typed this before... 😖 Please add `border-radius: 4px` here as well, so that all 4 corners are rounded when the header is hidden.